svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -443,6 +443,19 @@ def load_from_npz(self):
         # if self.npz_file is not None:
         #     self.npz_file.close()
 
+    def get_investigation_time(self):
+        try:
+            investigation_time = self.npz_file['investigation_time']
+        except KeyError:
+            investigation_time = None
```

Comment:
Please add a comment that this happens for outputs without investigation_time. For hazard outputs is must exists and cannot be None.